packages/operator/controllers/modelpackaging_controller_test.go

```diff
@@ -347,8 +347,8 @@ func (s *ModelPackagingControllerSuite) TestPackagingTimeout() {
 	mpNamespacedName := types.NamespacedName{Name: mp.Name, Namespace: mp.Namespace}
 	s.g.Expect(s.k8sClient.Get(context.TODO(), mpNamespacedName, mp)).ToNot(HaveOccurred())
 
-	tr := &tektonv1beta1.TaskRun{}
 	trKey := types.NamespacedName{Name: mp.Name, Namespace: testNamespace}
+	tr := s.getTektonPackagingTask(mp)
 	s.g.Expect(s.k8sClient.Get(context.TODO(), trKey, tr)).ToNot(HaveOccurred())
```

Comment:
```suggestion
      tr := s.getTektonPackagingTask(mp)
```
if compare it with the fix for `modeltraining_controller_test.go`, these lines also can be deleted.

Reply:
Agree. Fixed